.gitlab-ci.yml

```diff
@@ -5,34 +5,44 @@ stages:
   - build
   - test
 
-# disabled because cannot find -luv -lcap when building
+# disabled because gcc9 can't build net_if.hpp
 #build:linux:
 #  image: alpine:latest
 #  tags:
 #    - linux
 #  stage: build
 #  before_script:
-#    - apk add --update g++ make cmake linux-headers libcap-dev libuv-dev
+#    - apk add --update g++ make cmake linux-headers libcap-dev libuv-dev curl-dev git
 #  script:
-#    - make STATIC_LINK=ON
+#    - make STATIC_LINK=ON DOWNLOAD_SODIUM=ON
 #  artifacts:
 #    paths:
 #      - "lokinet"
 
 # we'll just try our travis set up for now
-#build:linux_release:
-#  image: ubuntu:xenial
-#  tags:
-#    - linux
-#  stage: build
-#  before_script:
-#    - apt-get update && apt-get install -y binutils-gold build-essential bsdmainutils ca-certificates cmake curl git libcap-dev libcurl4-openssl-dev libsodium-dev libuv1-dev ninja-build
-#  script:
-#    - DOWNLOAD_SODIUM=ON STATIC_LINK=OFF make release
-#  artifacts:
-#    paths:
-#      - "build/daemon/lokinet"
-#      - "build/daemon/lokinetctl"
+build:linux_release:
+  image: ubuntu:xenial
+  tags:
+    - linux
+  stage: build
+  before_script:
+    - apt-get update && apt-get install -y binutils-gold build-essential bsdmainutils ca-certificates cmake curl git libcap-dev pkg-config wget xz-utils libuv1-dev libsodium-dev libcurl4-openssl-dev
+  script:
+    - make static DOWNLOAD_SODIUM=ON
+    # alternatively: git describe --tag
+    - LOKINET_VERSION=$(./lokinet-static --version|sed s/lokinet-//)
+    - echo lokinet-linux-x64-v$LOKINET_VERSION
+    - mkdir lokinet-linux-x64-v$LOKINET_VERSION
+    - cp lokinet-bootstrap lokinet-linux-x64-v$LOKINET_VERSION
+    - cp build/daemon/lokinet lokinet-linux-x64-v$LOKINET_VERSION
+    - cp build/daemon/lokinetctl lokinet-linux-x64-v$LOKINET_VERSION
+    - cp linux-setup/readme.txt lokinet-linux-x64-v$LOKINET_VERSION
```

Comment:
Are these already root owned in the build?

Reply:
Yes:
```
$ tar tvf lokinet-linux-x64-v0.6.3-20d468e8e.tar.xz 
drwxr-xr-x  0 root   root        0 Jan 16 18:10 lokinet-linux-x64-v0.6.3-20d468e8e/
-rwxr-xr-x  0 root   root     1661 Jan 16 18:10 lokinet-linux-x64-v0.6.3-20d468e8e/lokinet-bootstrap
-rwxr-xr-x  0 root   root  6318176 Jan 16 18:10 lokinet-linux-x64-v0.6.3-20d468e8e/lokinet
-rwxr-xr-x  0 root   root  3446640 Jan 16 18:10 lokinet-linux-x64-v0.6.3-20d468e8e/lokinetctl
```